metals/src/main/scala/scala/meta/internal/pantsbuild/commands/CreateCommand.scala

```diff
@@ -39,19 +39,35 @@ object CreateCommand extends Command[CreateOptions]("create") {
     }
   }
   def run(create: CreateOptions, app: CliApp): Int = {
-    val name = create.actualName
-    Project.fromName(name, create.common) match {
-      case Some(value) =>
-        app.error(
-          s"can't create project named '${name}' because it already exists." +
-            s"\n\tTo refresh the project run: fastpass refresh ${name}"
-        )
-        1
-      case None =>
-        val project = Project.create(name, create.common, create.targets)
-        SharedCommand.interpretExport(
-          Export(project, create.open, app).copy(export = create.export)
-        )
+    val coursierBinary = create.coursierBinary.map(AbsolutePath(_))
+    val pathError = coursierBinary
+      .map { path =>
+        if (!path.isFile) {
```

Comment:
We can skip this validation. It's OK if the path does not exist. We only need this flag to allow testing fastpass in a CI environment that doesn't have access to internet and the bsp JSON files are only used by IntelliJ (not CI).